modules/widgets/authors.php

```diff
@@ -90,14 +90,27 @@ public function widget( $args, $instance ) {
 		 */
 		$excluded_authors = apply_filters( 'jetpack_widget_authors_exclude', $default_excluded_authors );
 
-		$authors = get_users(
+		/**
+		 * Filter the parameters of `get_users` call in the Widget Authors widget.
+		 *
+		 * See the following for `get_users` default arguments:
+		 * https://codex.wordpress.org/Function_Reference/get_users
+		 *
+		 * @module widgets
+		 *
+		 * @since 7.6.0
```

Comment:
```suggestion
		 * @since 7.7.0
```

We're past the code freeze for 7.6.0, so this is going to have to wait until the next release I am afraid :)